### ISSUE TITLE
Indentations (edit mode) - not accessible with screen readers

### ISSUE THREAD

**Issue by CarmenHilpert** [ckeditor/ckeditor5#19159]
## 📝 Provide detailed reproduction steps (if any)

1. Open https://ckeditor.com/docs/ckeditor5/latest/features/indent.html
2. Place cursor in the header line
4. Press the arrow down key

### ✔️ Expected result

- The screen reader should announce the indentation of the text

### ❌ Actual result

- Screen reader output is as it is with normal text

### ❓ Possible solution

 - Insert a text for screen readers only, e.g.
<p style="margin-left: 20px;">
   <span class="sr-only">Indented content start</span>
   This is the indented text in the paragraph.
  <span class="sr-only">Indented content end</span>
</p>

## 📃 Other details

* Browser: Edge 138.0.3351.65
* OS: Windows 11 (10.0.22631 Build 22631)
* Screenrader: JAWS 2024
* First affected CKEditor version: not shure when it started - seen in v44.3 and later
* Installed CKEditor plugins: see demo page

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.
